packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -168,10 +173,14 @@ const AppModals = ({
         ComponentToLoad = <NeedRealAccountForCashierModal />;
     }
     //TODO: Have to update this when cashier pop-up is done,this is just temporary
-    if (is_from_eu) {
+    if (is_verification_modal_visible) {
         ComponentToLoad = <VerificationModal />;
     }
 
+    if (is_verification_submitted) {
+        ComponentToLoad = <VerificationDocumentSubmitted />;
+    }
```

Comment:
Do we need this extra flag to display submitted modal?
can't we display this modal on submit event?

Reply:
We have to render this modal first only we can display out from the App. am I correct? 🤔

Reply:
This modal should be displayed only when POI/POA docs are submitted. right? 
cant we just  display the modal on submit ?